KPCTabsControl/DefaultTheme.swift

```diff
@@ -26,15 +26,15 @@ public struct DefaultTheme: Theme {
     private static var sharedBackgroundColor: NSColor { return NSColor(calibratedWhite: 0.95, alpha: 1.0) }
     private static var sharedHighlightedBackgroundColor: NSColor { return NSColor(calibratedWhite: 0.85, alpha: 1.0) }
 
-    struct DefaultTabButtonTheme: KPCTabsControl.TabButtonTheme {
+    private struct DefaultTabButtonTheme: KPCTabsControl.TabButtonTheme {
         
         var backgroundColor: NSColor { return DefaultTheme.sharedBackgroundColor }
         var borderColor: NSColor { return DefaultTheme.sharedBorderColor }
         var titleColor: NSColor { return NSColor.darkGrayColor() }
         var titleFont: NSFont { return NSFont.systemFontOfSize(13) }
     }
 
-    struct HighlightedTabButtonTheme: KPCTabsControl.TabButtonTheme {
+    private struct HighlightedTabButtonTheme: KPCTabsControl.TabButtonTheme {
```

Comment:
can we get rid of this variant completely, since there's no highlighting anymore?